sync_diff_inspector/diff.go

```diff
@@ -421,9 +421,9 @@ func (df *Diff) consume(ctx context.Context, rangeInfo *splitter.RangeInfo) bool
 	if rangeInfo.ChunkRange.Type == chunk.Empty {
 		dml.node.State = checkpoints.IgnoreState
 		// for tables that don't exist upstream or downstream
-		if !source.AllTableExist(tableDiff) {
-			upCount, _ := dbutil.GetRowCount(ctx, df.upstream.GetDB(), schema, table, "", nil)
-			downCount, _ := dbutil.GetRowCount(ctx, df.downstream.GetDB(), schema, table, "", nil)
+		if !common.AllTableExist(tableDiff.TableLack) {
+			upCount := df.upstream.GetCountAndCrc32(ctx, rangeInfo).Count
+			downCount := df.downstream.GetCountAndCrc32(ctx, rangeInfo).Count
```

Comment:
We only need count here. However, this function will also compute checksum. I'm afraid this will affect the efficiency.

Reply:
Got it. I defined a new interface `GetCountForLackTable` in the next commit([34293bb](https://github.com/pingcap/tidb-tools/pull/693/commits/34293bb67f87a72041106503cf6bfcebee520d9c)).